src/NzbDrone.Core/ImportLists/PassThePopcorn/PassThePopcornSettings.cs

```diff
@@ -0,0 +1,44 @@
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.ImportLists.PassThePopcorn;
+
+public class PassThePopcornCollectionSettingsValidator : AbstractValidator<PassThePopcornCollectionSettings>
+{
+    public PassThePopcornCollectionSettingsValidator()
+    {
+        RuleFor(c => c.BaseUrl).ValidRootUrl();
+        RuleFor(c => c.APIUser).NotEmpty();
+        RuleFor(c => c.APIKey).NotEmpty();
+        RuleFor(c => c.Id).GreaterThan(0);
+    }
+}
+
+public class PassThePopcornCollectionSettings : IProviderConfig
+{
+    private static readonly PassThePopcornCollectionSettingsValidator Validator = new PassThePopcornCollectionSettingsValidator();
+
+    public PassThePopcornCollectionSettings()
+    {
+        BaseUrl = "https://passthepopcorn.me";
+    }
+
+    [FieldDefinition(0, Label = "URL", Advanced = true, HelpText = "Don't change this unless you know what you're doing, since your API key will be sent to this host.")]
+    public string BaseUrl { get; set; }
+
+    [FieldDefinition(1, Label = "APIUser", Privacy = PrivacyLevel.UserName)]
+    public string APIUser { get; set; }
+
+    [FieldDefinition(2, Label = "APIKey", Privacy = PrivacyLevel.Password)]
+    public string APIKey { get; set; }
```

Comment:
@Qstick Just to confirm: these fields should be named `ApiKey` in line with the other fields (`BaseUrl`, `Id`, etc.), right?

Reply:
Yes